coalib/collecting/Collectors.py

```diff
@@ -452,7 +452,9 @@ def collect_registered_bears_dirs(entrypoint):
     for ep in pkg_resources.iter_entry_points(entrypoint):
         registered_package = None
         try:
-            registered_package = ep.load()
+            # backwards compat with old setuptools that do not provide Entrypoint.resolve
+            meth = getattr(ep, 'resolve', functools.partial(ep.load, require=False))
```

Comment:
Hi, if you want to contribute to coala, the best way is to follow coala's contribution guide. http://api.coala.io/en/latest/Developers/Newcomers_Guide.html That is, you'd better create an issue first, describe what improvement you want, get assigned and then work on it.
btw load() function doesn't have argument 'require'.

Reply:
`Signature: pr.EntryPoint.load(self, require=True, *args, **kwargs)`

Reply:
I saw in CI that an ancient version does not support it. Fallback wasn't worth it that deeply, so I just fallback once to a call that does require as it used to (therefore is extremely slower)